main.tf

```
@@ -57,6 +57,20 @@ resource "aws_rds_cluster_instance" "writer" {
   monitoring_role_arn = var.enhanced_monitoring ? aws_iam_role.rds_enhanced_monitoring[0].arn : null
 }
 
+
+resource "aws_rds_cluster_instance" "reader" {
+  count               = var.reader_instance_type ? 0 : 1
```

Can the Terraform ternary convert null to a falsy value or do we need to do this?
```suggestion
  count               = var.reader_instance_type == null ? 0 : 1
```

---

We chose this way to write it since the same way was used for 
```
monitoring_interval = var.enhanced_monitoring ? 60 : 0
```

---

The question was due to me not knowing if it actually handled it. Since it seems to work in other places I'm fine with that way of writing it. 😊